subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -486,15 +520,15 @@ int nrf_cloud_decode_data_endpoint(const struct nrf_cloud_data *input,
 		}
 	}
 
-	cJSON *tx_obj = json_object_decode(topic_obj, JSON_KEY_D2C);
+	cJSON *tx_obj = json_object_decode(topic_obj, DEVICE_TO_CLOUD_SUFFIX);
```

Comment:
how about using the name ‘JSON_KEY_DEVICE_TO_CLOUD’?
and similar for c2d.

Reply:
Sure. Prior to a recent rebase and merge conflict resolution I had to do, his was hard coded to "d2c".